examples/nodejs/token-vending-machine/README.md

```diff
@@ -69,7 +69,7 @@ You should see an output like:
 ```
 
 You can also pass a `tokenId` as a query string to your `curl` command. This is particularly useful to add context
-to your token, that can be viewed as a part of Momento [topic's](https://www.gomomento.com/services/topics) subscription messages.
+to your token, that can be viewed as a part of [Momento Topic's](https://www.gomomento.com/services/topics) subscription messages.
```

Comment:
Alternate phrasing suggestion for clarity:

> You can also pass a `tokenId` as a query string to your `curl` command to add context to your token. This can be particularly useful when using [Momento Topics](https://www.gomomento.com/services/topics) as the `tokenId` can be retrieved from subscription messages, allowing your application to distinguish between different subscribers.

Reply:
Love it!